components/AgreementCard/styles.js

```diff
@@ -1,4 +1,5 @@
 import styled from 'styled-components';
+import CounterAlert from 'components/CounterAlert';
```

Comment:
Vale, podríamos prevenir el StyledStyledCounterAlert si aquí importáramos como `import CounterAlert as CounterAlertBase from 'components/CounterAlert';` y luego `const CounterAlert = styled(CounterAlertBase)`. Pero si piensas que ya es mucho cambio preferible dejarlo como está.

Reply:
Dale, ahí lo cambio.

Reply:
changed in [247330f](https://github.com/openpolitica/proclama-preliminar/pull/100/commits/247330fbd2c4171c9cca93d48b794e59bd9403fa)